manuscript/ha-docker-swarm/traefik.md

```diff
@@ -133,8 +131,7 @@ version: "3"
 
 services:
   traefik:
-    image: traefik
-    command: --web --docker --docker.swarmmode --docker.watch --docker.domain=example.com --logLevel=DEBUG
+    image: traefik:v2.2
```

Comment:
```suggestion
    image: traefik:v2
```

Reply:
Always better to stick to specific versions. Syntax could change.

Reply:
Actually, thinking about *this* one, it should be okay.